internal/commands/ize.go

```diff
@@ -87,11 +88,14 @@ func init() {
 	rootCmd.PersistentFlags().String("terraform-version", "", "set terraform-version")
 	rootCmd.PersistentFlags().String("prefer-runtime", "native", "set prefer runtime (native or docker)")
 	rootCmd.Flags().StringP("tag", "t", "", "set tag")
-	viper.BindPFlags(rootCmd.PersistentFlags())
+	// viper.BindPFlags(rootCmd.PersistentFlags())
 	rootCmd.PersistentFlags().VisitAll(func(f *pflag.Flag) {
-		viper.BindPFlag(strings.ReplaceAll(f.Name, "_", "-"), rootCmd.PersistentFlags().Lookup(f.Name))
+		viper.BindPFlag(strings.ReplaceAll(f.Name, "-", "_"), rootCmd.PersistentFlags().Lookup(f.Name))
```

Comment:
@psihachina Can you please give more info on why this change is required?
What tests have we done to validate that ENV, flags and configs are still working?

Reply:
@AutomationD 
It was wrong. We need the flags `-` to be replaced with `_`. Otherwise, our flags are not bound to the identical value with `_` and are duplicated

For examples:
Old output all settings:
`
map[app:map[goblin:map[path:./projects/goblin type:ecs] squibby:map[path:./projects/squibby type:ecs]] apps_path:/home/psih/go/src/github.com/psihachina/ize/examples/ecs-apps-monorepo/projects aws-profile:test aws-region: aws_profile:default aws_region:us-east-1 config-file: env:test-ecs-apps env_dir:/home/psih/go/src/github.com/psihachina/ize/examples/ecs-apps-monorepo/.ize/env/test-ecs-apps home:/home/psih infra_dir:/home/psih/go/src/github.com/psihachina/ize/examples/ecs-apps-monorepo/.ize log-level: namespace:testnut plain-text:false prefer-runtime:native prefer_runtime:native root_dir:/home/psih/go/src/github.com/psihachina/ize/examples/ecs-apps-monorepo tag:de6c646 terraform-version: terraform_version:1.1.7 tf_log:]
`

With this change:
`
map[app:map[goblin:map[path:./projects/goblin type:ecs] squibby:map[path:./projects/squibby type:ecs]] aws_profile:test aws_region:us-east-1 config_file: env:test-ecs-apps env_dir:/home/psih/go/src/github.com/psihachina/ize/examples/ecs-apps-monorepo/.ize/env/test-ecs-apps home:/home/psih infra_dir:/home/psih/go/src/github.com/psihachina/ize/examples/ecs-apps-monorepo/.ize log_level: namespace:testnut plain_text:false prefer_runtime:native projects_path:/home/psih/go/src/github.com/psihachina/ize/examples/ecs-apps-monorepo/projects root_dir:/home/psih/go/src/github.com/psihachina/ize/examples/ecs-apps-monorepo tag:de6c646 terraform_version:1.1.7 tf_log:]
`